src/components/fin/metricSlider/heading/HeadingButtons.tsx

```diff
@@ -30,20 +35,52 @@ type Props = {
 }
 export default function HeadingButtons({ size = 'md', endPage }: Props) {
   const { sharedUrl } = useEndPageSharedUrl()
+  const [shouldDisplayConfirmMessage, setShouldDisplayConfirmMessage] =
+    useState(false)
+
+  const { t } = useClientTranslation()
+
+  const timeoutRef = useRef<NodeJS.Timeout | null>(null)
 
   const handleShare = async () => {
-    if (navigator?.share) {
+    if (navigator?.share && window.innerWidth < 768) {
```

Comment:
Magic number

Reply:
Imperfect but if it works.. Do you have another idea ?

Reply:
Just make 768 a constant that explain why. e.g.

```
const MAX_SCREEN_WIDTH_FOR_SHARE = 768 // mobile

if (navigator?.share && window.innerWidth < MAX_SCREEN_WIDTH_FOR_SHARE) {
  // share
}
```

It is OK to have magic numbers. It is better to explain why

Reply:
```suggestion
const MAX_SCREEN_WIDTH_FOR_SHARE = 768 // mobile

if (navigator?.share && window.innerWidth < MAX_SCREEN_WIDTH_FOR_SHARE) {
  // share
}
```

Reply:
done !